File: Minigames.Domain/Entities/PlayerGameSummary.cs
namespace Minigames.Domain.Entities;

public class PlayerGameSummary
{
   public HangmanGameResult HangmanGameResult { get; private set; }

   public FormulaGameResult FormulaGameResult { get; private set; }

    public PlayerGameSummary()
    {
        HangmanGameResult = new HangmanGameResult();
        FormulaGameResult = new FormulaGameResult();
    }
}

File: minigames-ui/src/app/models/player-game-summary.ts

import { FormulaGameResult } from './formula-game-result';
import { HangmanGameResult } from './hangman-game-result';

export interface PlayerGameSummary {
  playerId: number;
  playerName: string;
  formulaGameResult: FormulaGameResult;
  hangmanGameResult: HangmanGameResult;
}
